ckanext/security/controllers.py

```diff
@@ -53,7 +52,7 @@ def request_reset():
         # Later versions of CKAN core have fixed this behaviour, we default
         # to overriding with our own implementation but allow client to
         # disable if needed
-        if asbool(config.get(
+        if tk.asbool(config.get(
```

Comment:
What CKAN version makes `tk.asbool` available? Does CKAN 2.7 need a fallback?

Reply:
I'll look into that

Reply:
I went back as far as ckan 2.5.0 and toollkit had asbool on it even that far back :+1: